.github/workflows/setup-boilerplate.yml

```diff
@@ -4,14 +4,29 @@ on:
   workflow_dispatch:
     inputs:
       boilerplate:
-        type: string
+        type: choice
         description: Name of boilerplate
+        options:
```

Comment:
general: should add a note that when creating a new boilerplate should also update related challenges `setup-boilerplate` workflow

Reply:
Fair point. Do you mean in our internal docs somewhere so we don't forget?